train.py

```diff
@@ -30,7 +31,7 @@
     help="path to model to load (default: None)")
   parser.add_argument(
     "--model.type",
-    dest="model_type", type=str, default="realikun",
```

Comment:
Avoid changing default args for testing when possible. Quick check for this is to just run a diff on the train script